### ISSUE TITLE
Response from hub

### ISSUE THREAD

**Issue by Tekki** [skx/Device-Osram-Lightify#9]
Did you ever analyze the response of the hub? You read it at the end of the setters, but don't use it (the documentation on the side of Osram seems to be +/- zero).
At the moment we have the following:
```
say $light->status; # off
$light->set_on;
say $light->status; # off
```

**Comment by skx** [comment]
No, I never did.  it is something that _should_ be done, but at the time I set this up I was just pleased that it worked at all!

I've accepted your merge-request for the checking, and will test it myself now.  If it looks good I guess this issue can be closed and a new release made at the weekend.

**Comment by Tekki** [comment]
Please no stress this time.
One thing to notice is that as soon as we check for errors there actually will be errors. Every now and then the hub fails to execute a command. This is why I slowed down the live test, but I'm not sure if the speed is the reason.
I wonder why the hub doesn't complain if the same token is sent again (you use `time` to generate it, so all requests in the same second have the same token). The question is if it is a session token as you call it or a request token.

Btw, I was pleased too that your code worked from beginning. 👍
